<launch>
  <!-- Turtlebot -->
  <arg name="serialport"        default="$(env TURTLEBOT_SERIAL_PORT)"  doc="used by create to configure the port it is connected on [/dev/ttyUSB0, /dev/ttyS0]"/>
  <arg name="robot_name"        default="robot1"/>

  <group ns="$(arg robot_name)">

    <include file="$(find multi_turtlebot_bringup)/launch/includes/mobile_base.launch.xml">
      <arg name="serialport" value="$(arg serialport)" />
      <arg name="robot_name" value="$(arg robot_name)"/>
    </include>
    </include>
  </group>

</launch>
